lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -1457,6 +1457,20 @@ discouraged, to use the value specified as `truevalue` or `falsevalue`.
 Data input parameters can be given as a file name. The file should exist in the
 `test-data` folder. Multiple files can be specified as comma separated list.
 
+Note that it is possible to use zipped (gz, bz2) files in the `test-data` folder
+also for unzipped data types which is useful for large test data files.
+Galaxy will unzip these files on upload.
+
+```xml
+<param value="test.bed.gz" ftype="bed">
```

Comment:
Curiosity: will Galaxy unzip also for a datatype with `auto_compressed_types`, e.g.:
```xml
<param value="test.fastq.gz" ftype="fastq" />
```

Reply:
Yes, it does: https://github.com/planemo-autoupdate/tools-iuc/blob/e0944ef5a51ee0b4d2cc43ffa157a1ac3bbd3458/tools/read_it_and_keep/read-it-and-keep.xml#L121 (https://github.com/galaxyproject/tools-iuc/pull/4384)

Will add a test for this as well.